spec-main/api-protocol-spec.ts

```diff
@@ -565,21 +513,21 @@ describe('protocol module', () => {
       after(() => server.close())
       await server.listen(0, '127.0.0.1')
 
-      const port = server.address().port
+      const port = (server.address() as AddressInfo).port
       const url = `http://127.0.0.1:${port}`
-      const handler = (request, callback) => {
+      await interceptHttpProtocol('http', (request, callback) => {
+        console.log(request)
         const data = {
           url: url,
           method: 'POST',
           uploadData: {
             contentType: 'application/x-www-form-urlencoded',
-            data: request.uploadData[0].bytes.toString()
+            bytes: request.uploadData[0].bytes
           },
-          session: null
+          session: null as any
```

Comment:
Why does this need to be casted to any? If it's an allowed value lets update the docs

Reply:
looks like it doesn't /shrug

this test is disabled, btw. not sure why.